.expeditor/verify_private.pipeline.yml

```diff
@@ -61,7 +61,10 @@ steps:
         HAB_STUDIO_SECRET_GITHUB_TOKEN:
           account: github
           field: token
-      executor:
+        GITHUB_TOKEN:
+          account: github
+          field: token
+       executor:
```

Comment:
Looks like a YAML whitespace issue: need one less space indent

Reply:
Jinx! (thank you -- just pushed a change)